hal/halx86/mp/ioapic.c

```diff
@@ -106,7 +106,7 @@ IRQPolarity(ULONG idx)
 		  break;
 
 	       default:
-		  DPRINT("Broken BIOS!!\n");
+                  DPRINT1("Broken BIOS\n");
```

Comment:
you changed from tab to space indentation here. Would it make sense to treat the neighbor lines / or the whole function as well? To avoid mixed usage.

Reply:
I concur this file needs re-formatting, but not in this PR.

Reply:
Accepted to do it in a separate PR.